### PR TITLE
[APM2-131] Added new payment method OCBC PAO

### PR DESCRIPTION
**Pull request by tookaa2**
Added mobile banking OCBC PAO
https://omise.atlassian.net/browse/APM2-13
Screen Shot 2564-11-04 at 23 18 27
Screen Shot 2564-11-05 at 10 29 19

### CODE REVIEW

**Comment by kan98** [comment]
make sure it doesn't conflict with https://github.com/omise/omise-ios/pull/149 the row numbers for the mobile banking payment methods seems to be the same

**Comment by tookaa2** [comment]
> make sure it doesn't conflict with #149 the row numbers for the mobile banking payment methods seems to be the same

@kan98 I'll merge this PR in after #149 khub 🙏

File: OmiseSDK/Compatibility/OmiseCapability.swift [hunk 1]
```diff
@@ -87,8 +87,13 @@ class __OmiseCapabilitySourceBackendPayment: __OmiseCapabilityBackendPayment {
     static let eContextSourceBackendPayment =
     __OmiseCapabilitySourceBackendPayment(sourceType: OMSSourceTypeValue.eContext)
     
-    static let mobileBankingSourceBackendPayment =
-        __OmiseCapabilitySourceBackendPayment(sourceType: OMSSourceTypeValue.mobileBankingSCB)
+//    static let mobileBankingSourceBackendPayment =
+//        __OmiseCapabilitySourceBackendPayment(sourceType: OMSSourceTypeValue.mobileBankingSCB)
```
Comment (tobias90) [inline OmiseSDK/Compatibility/OmiseCapability.swift:91]: Can we remove this?

File: OmiseSDK/Compatibility/OmiseSource.swift [hunk 2]
```diff
@@ -315,7 +315,10 @@ public class __SourceMobileBankingPayment: __SourcePaymentInformation {
 
     /// Payment Information for a SCB Mobile Banking Payment
     public static let scbMobileBankingPayment = __SourceMobileBankingPayment(type: OMSSourceTypeValue.mobileBankingSCB)!
-
+    
+    /// Payment Information for a OCBC PAO Mobile Banking Payment
+        public static let ocbcPaoMobileBankingPayment = __SourceMobileBankingPayment(type: OMSSourceTypeValue.mobileBankingOCBCPAO)!
```
Comment (tobias90) [inline OmiseSDK/Compatibility/OmiseSource.swift:320]:
```suggestion
    public static let ocbcPaoMobileBankingPayment = __SourceMobileBankingPayment(type: OMSSourceTypeValue.mobileBankingOCBCPAO)!
```

File: OmiseSDK/PaymentCreatorController.swift [hunk 3]
```diff
@@ -535,6 +535,7 @@ extension PaymentCreatorController {
         .internetBankingSCB,
         .internetBankingBBL,
         .mobileBankingSCB,
+        .mobileBankingOCBCPAO,
```
Comment (tobias90) [inline OmiseSDK/PaymentCreatorController.swift:538]: Is OCBC PAO not a payment method for Singapore? Then we should add it to the presets for Singapore.

**Review by tobias90 — APPROVED** [review]

**Review by kan98 — APPROVED** [review]

**Pull request merged**